TLM/TLM/UI/SubTools/LaneConnectorTool.cs

```diff
@@ -402,45 +516,80 @@ private class LaneEnd {
                             bool highlightMarker = laneEnd == selectedLaneEnd || markerIsHovered;
                             laneEnd.RenderOverlay(cameraInfo, color, highlightMarker, true, group);
                         }
-                    } // if drawMarker
-                } // end foreach lanemarker in node markers
-            } // end for node in all nodes
+                    }
+                }
+            }
+        }
+        private void HandleStateMachine() {
```

Comment:
Can leave a TODO comment here to rewrite this later using generic StateMachine that we have in new tools.

Reply:
is there an issue to track this? to do without tracking issues is not a good idea.

Reply:
what is the issue for new subtools?

Reply:
There isn't an issue, it works don't touch it, but i have a state machine which i will use when upgrading this tool code